raiden/network/proxies/secret_registry.py

```diff
@@ -47,7 +46,7 @@
 class SecretRegistry:
     def __init__(self, jsonrpc_client, secret_registry_address, contract_manager: ContractManager):
         if not is_binary_address(secret_registry_address):
-            raise InvalidAddress("Expected binary address format for secret registry")
+            raise ValueError("Expected binary address format for secret registry")
```

Comment:
Shouldn't this be `InvalidBinaryAddress`

Reply:
@hackaugusto do you agree to the comment?

Reply:
Not really, I'm trying to split exceptions by domain, and I wanted to use the `InvalidBinaryAddress` and `InvalidChecksummedAddress` for user input validation. That piece would be an internal error.

Do you think it would beneficial for our tests to fail with an `InvalidBinaryAddress` exception?